tests/sentry/issues/auto_source_code_config/test_code_mapping.py

```diff
@@ -303,52 +301,76 @@ def test_list_file_matches_multiple(self) -> None:
         assert matches == expected_matches
 
     def test_find_roots_starts_with_period_slash(self) -> None:
-        stacktrace_root, source_path = find_roots("./app/", "static/app/")
```

Comment:
I don't know why I originally wrote the tests without a filename at the end of the path. It does not make sense to me.

From looking at the calls, it is obvious that two full paths are used rather than base dirs:
https://github.com/getsentry/sentry/blob/ed6ca2038d8f3fe68c2debd6c318ddaada86acab/src/sentry/issues/auto_source_code_config/code_mapping.py#L159-L163